safe-global/safe-wallet-web
# [Counterfactual] Disable notifications and message signing

## What it solves

Part of #3156

## How this PR fixes it

- Disables the Notifications Banner for newly created counterfactual safes
- Disables the Button on the Notifications settings page
- Hides counterfactual safes from the Global notifications page
- Disables the sign message flow for counterfactual safes

## How to test it

1. Create a new counterfactual safe
2. Observe no Popup saying "Enable all"
3. Go to the Notification settings
4. Observe a disabled button
5. Go to the global notification preferences
6. Observe the safe is not showing up in the list
7. Connect your safe with an app to create an off-chain message (e.g. opensea)
8. Observe an error message in the modal and the Sign button is disabled

## Screenshots
<img width="851" alt="Screenshot 2024-02-12 at 17 57 40" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/b9ba4feb-ca3c-4d4d-902f-792ab3ed1c91">
<img width="981" alt="Screenshot 2024-02-12 at 17 57 55" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/d641d1c4-a4af-43c7-acc5-d976bb8b74ab">
<img width="707" alt="Screenshot 2024-02-13 at 17 25 59" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/7c7ec69e-6022-493a-a6eb-dea91e310018">

## Checklist
